### DISCUSSION TITLE
用大会员提前购的已经死去 没看notice

### DISCUSSION THREAD

**Discussion by stukt18** [mikumifa/biliTickerBuy#261] [聊天专区]
只能等下周了

**Comment by 1258448151** [comment]
我也没看到...等下一周吧

**Comment by stukt18** [comment]
![image](https://github.com/mikumifa/biliTickerBuy/assets/99709480/73e6453a-ff60-4ba2-b11b-d43717a33625)

**Reply by stukt18** [reply]
5s过验证然后直接死去
